sass/styles.scss

```diff
@@ -0,0 +1,130 @@
+@use 'colors';
+@import 'components';
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+body {
+	background-color: colors.$dark;
+	overflow: hidden;
+	height: 100vh;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.bar {
+	color: colors.$dark;
+	padding: 16px;
+}
+
+.health {
+	padding: 16px;
+}
+
+.gameBoy {
+	position: relative;
+	min-width: 420px;
+	max-width: 420px;
+	background: colors.$yellow;
+	border-radius: 10px;
+	margin: 0 auto;
+	height: 657px;
+	display: flex;
+	justify-content: center;
+
+	.main-container {
+		margin-top: 54px;
+		height: 477px;
+		width: 100%;
+		margin-left: 6px;
+		margin-right: 6px;
+		border: 10px solid colors.$timber;
+
+		.inside__container {
+			height: 100%;
+			border: 10px solid #e7f6fd;
+			background-color: colors.$lightest-blue;
+			display: flex;
+
+			justify-content: center;
+		}
+
+		.rectangle {
+			width: 100%;
+			display: flex;
+			flex-direction: column;
+			margin: 10px;
+
+			&__parameters {
+				display: flex;
+				flex-direction: row;
+        gap: 10px;
+        line-height: 60px;
+        justify-content: space-between;
+
+				.parameter {
+					line-height: 40px;
+          display: flex;
+          gap:10px;
+          margin-bottom: 10px;
+
+					span {
+						width: 25px;
+						height: 60px;
+						font-family: VT323;
+						font-size: 60px;
+						height: 40px;
+						color: colors.$plum;
+					}
+				}
+			}
+
+			&__tamago {
+				width: 100%;
+				height: 100%;
+				border: 7px solid colors.$blue;
+				display: flex;
+				align-items: center;
+				justify-content: center;
+				margin-bottom: 15px;
+				&--svg {
+					position: relative;
+					&::after {
+						position: absolute;
+						bottom: 0;
```

Comment:
**nitpick:** Looks like you are missing formatting here - I would suggest you to install prettier in your project: https://prettier.io/